src/shadowbox/docker/Dockerfile

```diff
@@ -38,8 +38,15 @@ WORKDIR /root/shadowbox
 
 RUN mkdir bin && curl -SsL https://github.com/Jigsaw-Code/outline-ss-server/releases/download/v${SS_VERSION}/outline-ss-server_${SS_VERSION}_linux_x86_64.tar.gz | tar xz -C bin outline-ss-server
 COPY third_party/prometheus/prometheus ./bin/
-COPY src/shadowbox/package.json .
+COPY package.json .
 COPY yarn.lock .
+
+# Copy yarn workspace dependencies so `yarn install` can reference them.
+# Note that shadowbox app is already built and only depends on outline-metrics-server types.
```

Comment:
I don't understand the type dependency. That's used by Typescript at build time only, not?

Reply:
I've removed this comment. Please see my explanation above.